src/ts/ConvivaAnalyticsTracker.ts

```diff
@@ -454,12 +512,22 @@ export class ConvivaAnalyticsTracker {
         Conviva.SystemSettings.LogLevel.ERROR,
       );
     }
+
+    this.setPlayerInfo();
+    // It's required to correctly track VST. There must be BUFFERING or STOPPED metric reported before PLAYING.
+    // In some cases BUFFERING does not fire before PLAYING, so we report STOPPED right after session initialization to cover all edge cases.
+    this.convivaVideoAnalytics.reportPlaybackMetric(Conviva.Constants.Playback.PLAYER_STATE, Conviva.Constants.PlayerState.STOPPED);
```

Comment:
On mobile we do that **before** we do the `convivaVideoAnalytics.setCallback` setup. I think the idea was that there is no race condition that `convivaVideoAnalytics.setCallback` sneaks in before we report the initial state. Don't know how the timing of the first `setCallback` call is on web.

Reply:
Updated the web implementation to match the mobile https://github.com/bitmovin/bitmovin-player-web-analytics-conviva/pull/115/commits/aaef75b2fcd850375d4e2f257f4e47696993e27e.,